.github/workflows/release-step-3.yml

```diff
@@ -151,6 +151,7 @@ jobs:
         run: ./scripts/docker-release/push_docker.sh "${{ env.RELEASE_VERSION }}"
 
   publish-aws-lambda:
+    if: false
```

Comment:
I guess this was not intentional?

Reply:
It was not intentional. Thank you for checking.

Reply:
```suggestion
```